crates/bevy_gizmos/src/animated/mod.rs

```diff
@@ -0,0 +1,164 @@
+//! A module for the [`AnimatedGizmos`] [`SystemParam`].
+
+use std::ops::{Deref, DerefMut};
+
+use bevy_color::Color;
+use bevy_ecs::system::{Res, SystemParam};
+use bevy_math::Vec3;
+use bevy_time::Time;
+
+use crate::prelude::{DefaultGizmoConfigGroup, GizmoConfigGroup, Gizmos};
+
+/// A [`SystemParam`] for drawing animated gizmos.
+///
+/// This is basically just a utility wrapper around [`Gizmos`], so for more details take a look
+/// at the docs there.
+///
+/// Note that you can still draw the regular non-animated geometries as with regular [`Gizmos`]
```

Comment:
```suggestion
/// To draw standard, non-animated immediate-mode geometries, use [`Gizmos`].
```

Reply:
Oh no, that would be misleading I think. That sounds a bit as if one should just use `Gizmos` additionally even though `AnimatedGizmos` impl `DerefMut` exactly to prevent the user from duplicating system params

Reply:
You can just use `animated_gizmos.line(start,end,color);` with this impl!

Thanks for fixing my broken english though 🙈